cluster-logging/overlays/moc/configmaps/curator.yaml

```diff
@@ -0,0 +1,117 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: curator
+data:
+  actions.yaml: >
+    # ---
+
+    # Remember, leave a key empty if there is no value.  None will be a string,
+
+    # not a Python "NoneType"
+
+    #
+
+    # Also remember that all examples have 'disable_action' set to True.  If you
+
+    # want to use this action as a template, be sure to set this to False after
+
+    # copying it.
+
+    # actions:
+
+    #   1:
+
+    #     action: delete_indices
+
+    #     description: >-
+
+    #       Delete .operations indices older than 30 days.
+
+    #       Ignore the error if the filter does not
+
+    #       result in an actionable list of indices (ignore_empty_list).
+
+    #       See
+    https://www.elastic.co/guide/en/elasticsearch/client/curator/5.2/ex_delete_indices.html
+
+    #     options:
+
+    #       # Swallow curator.exception.NoIndices exception
+
+    #       ignore_empty_list: True
+
+    #       # In seconds, default is 300
+
+    #       timeout_override: ${CURATOR_TIMEOUT}
+
+    #       # Don't swallow any other exceptions
+
+    #       continue_if_exception: False
+
+    #       # Optionally disable action, useful for debugging
+
+    #       disable_action: False
+
+    #     # All filters are bound by logical AND
+
+    #     filters:
+
+    #     - filtertype: pattern
+
+    #       kind: regex
+
+    #       value: '^\.operations\..*$'
+
+    #       exclude: False
+
+    #     - filtertype: age
+
+    #       # Parse timestamp from index name
+
+    #       source: name
+
+    #       direction: older
+
+    #       timestring: '%Y.%m.%d'
+
+    #       unit: days
+
+    #       unit_count: 30
+
+    #       exclude: False
+  config.yaml: |
+    # Logging example curator config file
+
+    uncomment and use this to override the defaults from env vars
```

Comment:
Can you remove this line, I believe it was meant to be a comment :)

Reply:
good catch :+1: 
updated